recipes/kaldi/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: kaldi
+  version: r7271.1a4dbf6
+
+source:
+  git_url: https://github.com/kaldi-asr/kaldi.git
+  git_rev: 1a4dbf6be118e4967cf4efd510811ef72cdfee3d
+  patches:
+    - conda-kaldi.patch
```

Comment:
Can you add a comment describing the patch? What is it for and/or if it was submitted upstream?